packages/cml/src/replaceCmlPlaceholders/replaceCmlPlaceholders.ts

```diff
@@ -77,7 +77,11 @@ const replacePropValues = (mdx: string, values: Record<string, string> = {}): st
       case 'boolean':
       case 'number':
         searchValue = match;
-        newValue = `{${propValue ?? defaultValue}}`;
+        if (propValue === undefined && (defaultValue === undefined || defaultValue === '')) {
```

Comment:
nit: can we simplify this to:

```suggestion
        if (propValue === undefined && !defaultValue) {
```